engine/Default/chat_sharing_processing.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+function error_on_page($message) {
```

Comment:
I'd prefer to use `errorOnPage` to be a bit more consistent, but I am also resigned to the fact that no matter what php isn't a particularly consistent language :(

Reply:
I thought that we were supposed to use camel/pascal case for classes and methods, and snake case for free functions. Or are `create_container`, `create_error`, etc. relics of an earlier standard?

Reply:
Yeah, they're relics, a lot of stuff in smr is from times long gone

Reply:
It'd be nice to use something like https://github.com/PHPCheckstyle/phpcheckstyle, but I fear that the initial push for compliance would be extremely burdensome.